### DISCUSSION TITLE
Bootstrap 5 Migration

### DISCUSSION THREAD

**Discussion by stevenlupo** [Megabit/Blazorise#3219] [Q&A] [answered]
I upgraded my project from Bootstrap 4 to 5 just to check it out. Did anyone notice that the `<small></small>` element no longer works? 

Also:

* all  links are now underlined.
* the effect of `<Table Narrow="true">` now has a different look.

Curious if this is expected. There are other seemingly minor alignment issues, but these are the biggest changes I've noticed so far.

**Comment by stsrki** [comment, marked as answer]
Yes, the changes are expected and they can also be seen in the Bootstrap 5 documentation https://getbootstrap.com/docs/5.1/content/tables/#small-tables

While we could make them look the same as BS4 it would go against the BS5 guidelines.

**Comment by stevenlupo** [comment]
But what about the `<small></small>` element no longer working? That seems odd to me.

I'll explore it more.

**Reply by stsrki** [reply]
`<small></small>` seems to be working fine.

![image](https://user-images.githubusercontent.com/900302/145250753-fe81cbc2-d54e-445c-bd40-5b3b5ccaa568.png)
